tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -201,6 +202,37 @@ def test_fn_to_docker_container(self, ondemand_aws_cluster):
     ####################################################################################################
     # Status tests
     ####################################################################################################
+
+    @pytest.mark.level("minimal")
+    @pytest.mark.clustertest
+    def test_status_scheduler_basic_flow(self, cluster):
+
+        cluster_uri = rh.globals.rns_client.format_rns_address(cluster.rns_address)
+        api_server_url = rh.globals.rns_client.api_server_url
+        headers = rh.globals.rns_client.request_headers()
+
+        get_status_data_resp = requests.get(
+            f"{api_server_url}/resource/{cluster_uri}/cluster/status?limit=1",
+            headers=headers,
+        )
+
+        assert get_status_data_resp.status_code == 200
+        resp_data = get_status_data_resp.json().get("data")[0]
+        assert resp_data.get("status") == ResourceServerStatus.running
+
+        # check that the status was updated in the DB in the DEFAULT_STATUS_CHECK_INTERVAL minute(s),
+        # cause the status periodic check runs every DEFAULT_STATUS_CHECK_INTERVAL minute(s).
+        status_updated_at = datetime.datetime.fromisoformat(
```

Comment:
this feels more like a den test (that the DB was actually updated), what's important in OSS is that the status was collected as part of the periodic cluster check and actually sent off to den (with a 200).. maybe just confirming in the cluster logs tghat the ping succeeded would work. wdyt?